kits/find-your-hotel/.env.example

```
@@ -0,0 +1,5 @@
+LAMATIC_PROJECT_ENDPOINT= Your Lamatic GraphQL endpoint
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)